src/commands/ingress/IngressPromptStep.ts

```diff
@@ -0,0 +1,39 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { AzureWizardExecuteStep, AzureWizardPromptStep, IWizardOptions } from "@microsoft/vscode-azext-utils";
+import { localize } from "../../utils/localize";
+import { IngressContext } from "./IngressContext";
+import { DisableIngressStep } from "./disableIngress/DisableIngressStep";
+import { TargetPortInputStep } from "./editTargetPort/TargetPortInputStep";
+import { EnableIngressStep } from "./enableIngress/EnableIngressStep";
+import { IngressVisibilityStep } from "./enableIngress/IngressVisibilityStep";
+
+export class IngressPromptStep extends AzureWizardPromptStep<IngressContext> {
+    public async prompt(context: IngressContext): Promise<void> {
+        context.enableIngress = (await context.ui.showQuickPick([{ label: localize('enable', 'Enable'), data: true }, { label: localize('disable', 'Disable'), data: false }],
```

Comment:
Most of this logic was repurposed from the old `EnableIngressStep`... I ended up renaming the step and expanding the behavior of the subWizard.